cmd/mavenExecuteStaticCodeChecks_test.go

```diff
@@ -120,3 +120,16 @@ func TestGetSpotBugsMavenParameters(t *testing.T) {
 		assert.Equal(t, &expected, getSpotBugsMavenParameters(&config))
 	})
 }
+
+type mavenStaticCodeChecksTestUtilsBundle struct {
+	*mock.ExecMockRunner
+	*mock.FilesMock
+}
+
+func newMavenStaticCodeChecksUtilsBundle() mavenStaticCodeChecksTestUtilsBundle {
```

Comment:
Can't provide a suggestion because it needs changing in multiple places... but I'd vote for reflecting the name of the struct in the function, i.e. `newMavenStaticCodeChecksTestUtilsBundle()`.